tests/config/nextflow.config

```
@@ -14,6 +14,8 @@ process {
 if ("$PROFILE" == "singularity") {
     singularity.enabled = true
     singularity.autoMounts = true
+    wave.enabled = true
+    wave.strategy = ['dockerfile', 'conda', 'container']
```

```suggestion
    wave.enabled = true
    wave.strategy = ['conda', 'container']
    wave.freeze = true
    wave.build.repository = 'docker.io/emiller88/modules'
```

Singularity native requires the *freeze* mode, which in turn requires also the target build repository